tests/TestCase.php

```diff
@@ -24,43 +24,4 @@
 namespace OCA\AnnouncementCenter\Tests;
 
 abstract class TestCase extends \Test\TestCase {
-	/** @var array */
-	protected $services = [];
-
-	/**
-	 * @param string $name
-	 * @param mixed $newService
-	 * @return bool
-	 */
-	public function overwriteService($name, $newService) {
```

Comment:
I added it to `\Test\Testcase` directly months ago, so yeah useless overwriting.